indico/modules/events/reminders/models/reminders.py

```diff
@@ -143,6 +151,40 @@ def all_recipients(self):
         recipients = set(self.recipients)
         if self.send_to_participants:
             recipients.update(reg.email for reg in Registration.get_all_for_event(self.event))
+
+        if self.send_to_speakers:
+            event_person_emails = (person_link.email for person_link in self.event.person_links)
```

Comment:
This is a generator, but below you iterate it twice - the second time it's always empty. Make it a set comprehension instead

```suggestion
            event_person_emails = {person_link.email for person_link in self.event.person_links}
```

Reply:
actually, you can already do `recipients.update(...)` here, no need to assign it separately